index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/style/style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header class="flex justify-between" id="header">
+    <p class="self-center" id="myLogo"><a href="#">My Logo</a></p>
```

Comment:
I can see you made some effort towards implementing the change requested by the previous reviewer👍 but for semantic purposes, you don't need to wrap your logo inside a `p` tag😉 kindly take the `p` tag and just wrap your logo in an `a` tag.